source/elements/oneDAL/source/data_management/index.rst

```diff
@@ -1,7 +1,13 @@
+.. _data_management:
+
 ===============
 Data management
 ===============
 
+The Data Management component includes classes and utilities for data acquisition, initial preprocessing
+and normalization, for data conversion into numeric formats done by one of supported Data Sources
```

Comment:
I think it's better like this:

- data conversion into numeric formats (performed by one of supported Data Sources)

Reply:
fixed